src/Mod/Draft/Resources/ui/preferences-drafttexts.ui

```diff
@@ -504,6 +513,43 @@ such as "Arial:Bold"</string>
         </item>
        </layout>
       </item>
+      <item>
+       <layout class="QHBoxLayout" name="horizontalLayout_10">
+        <item>
+         <widget class="QLabel" name="label_3">
+          <property name="text">
+           <string>Feet separator</string>
+          </property>
+         </widget>
+        </item>
+        <item>
+         <spacer name="horizontalSpacer_3">
+          <property name="orientation">
+           <enum>Qt::Horizontal</enum>
+          </property>
+          <property name="sizeHint" stdset="0">
+           <size>
+            <width>40</width>
+            <height>20</height>
+           </size>
+          </property>
+         </spacer>
+        </item>
+        <item>
+         <widget class="Gui::PrefLineEdit" name="lineEdit_2">
+          <property name="toolTip">
+           <string>Here you can set an optional, customized separator to apear between the feet and inches values on dimensions</string>
```

Comment:
Tooltip can perhaps be sorter:
'Optional separator to appear between the feet and inches values in dimensions'